dts/arc/quark_se_c1000_ss.dtsi

```diff
@@ -0,0 +1,53 @@
+#include "skeleton.dtsi"
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "snps,arcem4";
```

Comment:
is the cpu really em4?

Reply:
yes. That is what mentioned in data sheet of quark_se_c1000

Reply:
Can you point me at the data sheet for the quark_se_c1000?

Reply:
https://www.intel.com/content/dam/www/public/us/en/documents/datasheets/quark-c1000-datasheet.pdf.

In first page only it is mentioned "ARC EM4 DSP with floating point 
extensions"